### PR TITLE
[15.0][MIG] stock_quantity_history_location: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Supersede https://github.com/OCA/stock-logistics-reporting/pull/202

- Inherit the template to change the button name
- Modify action["display_name"] instead of action["name"]
- Add `attrs="{'invisible': [('location_id', '=', False)]}"`  to the 'Include child locations' field in its XML view definition
- Squash administrative commits to reduce the commit history

Cc @Tecnativa TT37324

ping @pedrobaeza @chienandalu @desdelinux @rousseldenis

### CODE REVIEW

File: stock_quantity_history_location/static/src/xml/inventory_report.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates>
+    <t t-extend="StockInventoryReport.Buttons">
+        <t t-jquery="button.o_button_at_date" t-operation="inner">
```
Comment (sergio-teruel) [inline stock_quantity_history_location/static/src/xml/inventory_report.xml:4]: We only must change the button description if we have group_stock_multi_locations security group.

**Review by sergio-teruel — APPROVED** [review]
Only little comment

File: stock_quantity_history_location/static/src/xml/inventory_report.xml (hunk 1 above)
Comment (ernestotejeda) [reply stock_quantity_history_location/static/src/xml/inventory_report.xml:4]: Right!

**Review by chienandalu — APPROVED** [review]

**Comment by ernestotejeda** [comment]
changes done @sergio-teruel

**Comment by desdelinux** [comment]
Hello @ernestotejeda , are you going to migrate this to 16.0?

**Comment by chienandalu** [comment]
Not soon @desdelinux . Feel free to do it :smile: (it shouldn't be much effort I think)

**Pull request merged**